guides/dev/issues_tracker/README.md

```diff
@@ -46,19 +46,24 @@ This is it. Once you log in, you will be ready to [report a CKEditor issue](http
 Every issue may be marked with the following labels:
 
 * Status labels &ndash; current issue status:
-    * <span style="background-color:#e6e6e6;padding:0 3px 0 3px">status:pending</span> &ndash; More information is needed to proceed with the issue.
     * <span style="background-color:#e6e6e6;padding:0 3px 0 3px">status:confirmed</span> &ndash; Issue confirmed by the development team.
+    * <span style="background-color:#e6e6e6;padding:0 3px 0 3px">status:pending</span> &ndash; More information is needed to proceed with the issue.    
 * Type labels - the type of the issue:
     * <span style="background-color:#b60205;color:#FFF;padding:0 3px 0 3px">type:bug</span> &ndash; A bug.
     * <span style="background-color:#1d76db;color:#FFF;padding:0 3px 0 3px">type:feature</span> &ndash; A feature request.
     * <span style="background-color:#fbca04;padding:0 3px 0 3px">type:task</span>&ndash; Any other issue (refactoring, typo fix, etc).
-* Resolution labels &ndash; how and why the issue was resolved:
-    * <span style="background-color:#444444;color:#FFF;padding:0 3px 0 3px">resolution:invalid</span> &ndash; Not a valid issue (not reproducible, 3rd-party bug, etc).
+* Resolution labels &ndash; how and why the issue was resolved:    
     * <span style="background-color:#444444;color:#FFF;padding:0 3px 0 3px">resolution:duplicate</span> &ndash; A duplicate of an already reported issue.
+    * <span style="background-color:#444444;color:#FFF;padding:0 3px 0 3px">resolution:expired</span> &ndash; Issue reporter did not provide enough information to reproduce the issue for a long time.
+    * <span style="background-color:#444444;color:#FFF;padding:0 3px 0 3px">resolution:invalid</span> &ndash; Not a valid issue (not reproducible, 3rd-party bug, etc).
```

Comment:
The 3rd party party bug has now dedicated resolution (upstream). You can add something like "wrong request type" in a sense that if some1 is asking support questions on GH tracker.